kolibri/plugins/pdf_viewer/assets/src/views/PdfPage.vue

```diff
@@ -84,35 +84,19 @@
       };
     },
     computed: {
-      actualHeight() {
```

Comment:
All these removed methods were only used to calculate`scaledWidth` and `scaledHeight`